b2share/config.py

```diff
@@ -1,76 +1,24 @@
-from invenio_base.config import PACKAGES as _PACKAGES
 
+# -*- coding: utf-8 -*-
 
-PACKAGES = [
-    "b2share.modules.*",
-    "b2share.base",
-    "invenio_oaiharvester"
-] + _PACKAGES
+"""B2Share base Invenio configuration."""
 
-PACKAGES_EXCLUDE = [
-    "b2share.modules.b2deposit"
-    'invenio_annotations',
-    'invenio_archiver',
-    'invenio_communities',
-    'invenio_linkbacks',
-    'invenio_multimedia',
-    'invenio_pages',
-    'invenio_deposit',
-]
-
-# all invenio modules because exclude does not accept wildcards
-invenio_modules = [
-    "invenio_collections",
-    'invenio_annotations',
-    'invenio_documentation',
-    'invenio_messages',
-    'invenio_records',
-    'invenio_authorprofiles',
-    'invenio_editor',
-    'invenio_oaiharvester',
-    'invenio_scheduler',
-    'invenio_authors',
-    'invenio_formatter',
-    'invenio_oairepository'
-    'invenio_tags',
-    'invenio_cloudconnector',
-    'invenio_groups',
-    'invenio_pages',
-    'invenio_uploader',
-    'invenio_comments',
-    'invenio_knowledge',
-    'invenio_pidstore',
-    'invenio_communities',
-    'invenio_linkbacks',
-    'invenio_previewer',
-    'invenio_access',
-    'invenio_accounts',
-    'invenio_deposit',
-    'invenio_oauth2server',
-    'invenio_oauthclient',
-    'invenio_search',
-    'invenio_workflows',
-]
+from __future__ import absolute_import, print_function
+import os
 
-# Exclude all Invenio views
-PACKAGES_VIEWS_EXCLUDE = invenio_modules
-# Exclude all Invenio REST APIs
-PACKAGES_RESTFUL_EXCLUDE = invenio_modules
 
+# Identity function for string extraction
+def _(x):
```

Comment:
I will remove it for the time being but I'm not sure we will be able to do without it. It is in fact already grepped in Invenio as it is the one used to generate automatically the translation files.